doc/classes/AudioStreamPlayback.xml

```diff
@@ -79,5 +79,18 @@
 				Overridable method. Called whenever the audio stream is mixed if the playback is active and [method AudioServer.set_enable_tagging_used_audio_streams] has been set to [code]true[/code]. Editor plugins may use this method to "tag" the current position along the audio stream and display it in a preview.
 			</description>
 		</method>
+		<method name="get_sample_playback" qualifiers="const" experimental="">
+			<return type="AudioSamplePlayback" />
+			<description>
+				Returns the meta class associated with this [AudioStreamPlayback] for playing back the audio sample of this stream.
+			</description>
+		</method>
+		<method name="set_sample_playback" experimental="">
+			<return type="void" />
+			<param index="0" name="playback_sample" type="AudioSamplePlayback" />
+			<description>
+				Associates the meta class to this [AudioStreamPlayback] for playing back the audio sample of this stream.
+			</description>
+		</method>
```

Comment:
It's not clear to me what "meta class" means in this context. It's also an Object instance and not a class itself, no?

Overall the subtleties of streams vs samples are likely hard to convey through just API descriptions, especially as most users won't know which is which and why they should care, so we'll probably want to cover this in a dedicated docs page if there's a need for some users to handle this themselves.